nvflare/apis/impl/job_def_manager.py

```diff
@@ -174,9 +177,11 @@ def _load_job_data_from_store(self, jid: str, temp_dir: str, fl_ctx: FLContext):
         unzip_all_from_bytes(data_bytes, job_id_dir)
         return job_id_dir
 
-    def get_content(self, jid: str, fl_ctx: FLContext) -> bytes:
+    def get_content(self, jid: str, fl_ctx: FLContext) -> Optional[bytes]:
         store = self._get_job_store(fl_ctx)
         stored_data = store.get_data(self.job_uri(jid))
+        if not stored_data:
```

Comment:
Instead of changing the storage code, should we just add a try catch block here?


An enhancement would be to change storage's RuntimeError to something like NVFlareObjectNotFoundError so we don't catch other unwanted situations

Reply:
when job not exist, return None make more sense than raise RuntimeError. Easier to distinguish with other real errors.